docs/en/setup/backend/endpoint-grouping-rules.md

```diff
@@ -1,15 +1,319 @@
+          description: Product name
+      required:
+        - id
+        - name
+    ProductDetails:
+      type: object
+      description: Product details
+      properties:
+        id:
+          type: integer
+          format: int64
+          description: Product id
+        name:
+          type: string
+          description: Product name
+        description:
+          type: string
+          description: Product description
+      required:
+        - id
+        - name
+    RelatedProducts:
+      type: object
+      description: Related Products
+      properties:
+        id:
+          type: integer
+          format: int32
+          description: Product id
+        relatedProducts:
+          type: array
+          description: List of related products
+          items:
+            $ref: "#/components/schemas/Product"
+
+
+```
+
+Here give some scenario we might use:
+1. Only set the `x-sw-service-name`, `x-sw-endpoint-name-match-rule` and `x-sw-endpoint-name-format` are default:
+``` yaml
+openapi: 3.0.0
+x-sw-service-name: serviceB
+
+info:
+  description: OpenAPI definition for SkyWalking test.
+  version: v2
+  title: Product API
+  ...
+```
+2. Set the `x-sw-service-name` , `x-sw-endpoint-name-match-rule` and `x-sw-endpoint-name-format` :
+``` yaml
+openapi: 3.0.0
+x-sw-service-name: serviceB
+x-sw-endpoint-name-match-rule: "<${METHOD}>:${PATH}"
+x-sw-endpoint-name-format: "<${METHOD}>:${PATH}"
+
+info:
+  description: OpenAPI definition for SkyWalking test.
+  version: v2
+  title: Product API
+  ...
+```
+3. Set the `x-sw-service-name` , `x-sw-endpoint-name-match-rule` and `x-sw-endpoint-name-format` :
+``` yaml
+openapi: 3.0.0
+x-sw-service-name: serviceB
+x-sw-endpoint-name-match-rule: "<${METHOD}>:${PATH}"
+x-sw-endpoint-name-format: "<${METHOD}>:${PATH}"
+
+info:
+  description: OpenAPI definition for SkyWalking test.
+  version: v2
+  title: Product API
+  ...
+```
+
+   | Incoming Endpiont | Incoming Service | x-sw-endpoint-name-match-rule | x-sw-endpoint-name-format | Matched | Grouping Result |
+   |-----|-----|-----|-----|-----|-----|
+   | GET:/products | serviceB | default | default | true | GET:/products |
+   | GET:/products/123 | serviceB | default |default |  true | GET:/products{id} |
+   | GET:/products/asia/cn | serviceB | default | default | true | GET:/products/{region}/{country} |
+   | GET:/products/123/abc/efg | serviceB | default |default |  false | GET:/products/123/abc/efg | 
+   | \<GET\>:/products/123 | serviceB | default | default | false | \<GET\>:/products/123|
+   | GET:/products/123 | serviceC | default | default | false | GET:/products/123 |
+   | \<GET\>:/products/123 | serviceB | \<\${METHOD}\>:\${PATH} | \<\${METHOD}>:\${PATH} | true | \<GET\>:/products/{id} |
+   | GET:/products/123 | serviceB | default | ${PATH}:\<\${METHOD}\> | true | /products/{id}:\<GET\> |
+   | /products/123:\<GET\> | serviceB | ${PATH}:\<\${METHOD}\> | default | true | GET:/products/{id} |
```

Comment:
I'd rather put these GET:/products, etc. etc. into `` so you don't need to escape the characters, which is error prone

Reply:
I am wondering the point of `\` too

Reply:
to escape the markdown keyword...  but if I put them into `` ,  the style (color) would differ from others.

Reply:
Could you put a screenshot here to demonstrate what will be seen?

Reply:
![image](https://user-images.githubusercontent.com/16773043/122528994-8490ad80-d04f-11eb-8c07-d7d4ddccea2e.png)
would like this

Reply:
@kezhenxu94 From the preview, it seems better to keep the current way?

Reply:
- `$`, `<`, and `>` has no special meanings in GitHub markdown, they may have in other extended markdown engine.

- You don't have consistent style in terms of quoting in backticks or not, for example you quote them here 
<img width="1065" alt="Screen Shot 2021-06-18 at 17 54 24" src="https://user-images.githubusercontent.com/15965696/122543360-46e75100-d05e-11eb-8d81-bbe8a1806e58.png">
but not in this table

- If you choose not to quote in `` but with `\` to escape those characters, see 

- There are cases when users may look into the source codes and copy the example with those `\`, which doesn't work.

Reply:
make sense, I have put all path/pattern string into ``.